src/rich-text/commands/index.ts

```diff
@@ -421,6 +474,43 @@ const headingDropdown = (schema: Schema) =>
         )
     );
 
+const overflowDropdown = (schema: Schema, options: CommonViewOptions) =>
+    makeMenuDropdown(
+        "Overflow",
```

Comment:
"Overflow" is not descriptive of the contents. Maybe "More" or "Misc"? What do other editors have for this text?

Reply:
Jira does "More formatting" so I went with that.